internal/supply/supply_test.go

```diff
@@ -0,0 +1,53 @@
+package supply
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const nativeNode = "https://zeus.mainnet.decentr.xyz"
+const ethNode = "" // nolint
```

Comment:
"https://mainnet.infura.io/v3/97828c3ed33f48d2bbc09b06aa9b7dc0" 
Also you can use it as default in the supply.erc20_node param